intelmq/conf/harmonization.conf

```diff
@@ -1,226 +1,357 @@
+        },
+        "source.ip": {
+            "description": "The ip observed to initiate the connection",
+            "type": "IPAddress"
+        },
+        "source.local_hostname": {
+            "description": "Some sources report a internal hostname within a NAT related to the name configured for a compromized system",
+            "type": "String"
+        },
+        "source.local_ip": {
+            "description": "Some sources report a internal (NATed) IP address related a compromized system. N.B. RFC1918 IPs are OK here.",
+            "type": "IPAddress"
+        },
+        "source.network": {
+            "description": "CIDR for an autonomous system. Also known as BGP prefix.",
+            "type": "IPNetwork"
+        },
+        "source.port": {
+            "description": "The port from which the connection originated.",
+            "regex": "^[0-9]{1,5}$",
+            "type": "Integer"
+        },
+        "source.registry": {
+            "ascii": true,
+            "description": "The IP registry a given ip address is allocated by.",
+            "type": "String"
+        },
+        "source.reverse_dns": {
+            "description": "Reverse DNS name acquired through a reverse DNS query on an IP address. N.B. Record types other than PTR records may also appear in the reverse DNS tree. Furthermore, unfortunately, there is no rule prohibiting people from writing anything in a PTR record. Even Javascript will work.",
+            "type": "FQDN"
+        },
+        "source.tor_node": {
+            "description": "If the source IP was a known tor node.",
+            "type": "Boolean"
+        },
+        "source.url": {
+            "description": "A URL denotes an IOC, which refers to a malicious resource, whose interpretation is defined by the abuse type. A URL with the abuse type phishing refers to a phishing resource.",
+            "type": "URL"
+        },
+        "status": {
+            "description": "Status of the malicious resource (phishing, dropzone, etc), e.g. online, offline.",
+            "type": "String"
+        },
+        "time.observation": {
+            "description": "The time a source bot saw the event. This timestamp becomes especially important should you perform your own attribution on a host DNS name for example. The mechanism to denote the attributed elements with reference to the source provided is detailed below in Reported Identity IOC.(ISO8660).",
+            "type": "DateTime"
+        },
+        "time.source": {
+            "description": "Time reported by a source. Some sources only report a date, which may be used here if there is no better observation.",
+            "type": "DateTime"
+        },
+        "user_agent": {
+            "description": "Some feeds report the user agent string used by the host to access a malicious resource, such as a command and control server.",
+            "type": "String"
+        }
+    },
+    "report": {
+        "feed.code": {
```

Comment:
---
Mobile


> On 28.08.2015, at 16:34, Tomás Lima <notifications@github.com> wrote:
> 
> In intelmq/conf/harmonization.conf:
> 
> > +        },
> > +        "time.observation": {
> > +            "description": "The time a source bot saw the event. This timestamp becomes especially important should you perform your own attribution on a host DNS name for example. The mechanism to denote the attributed elements with reference to the source provided is detailed below in Reported Identity IOC.(ISO8660).",
> > +            "type": "DateTime"
> > +        },
> > +        "time.source": {
> > +            "description": "Time reported by a source. Some sources only report a date, which may be used here if there is no better observation.",
> > +            "type": "DateTime"
> > +        },
> > +        "user_agent": {
> > +            "description": "Some feeds report the user agent string used by the host to access a malicious resource, such as a command and control server.",
> > +            "type": "String"
> > +        }
> > +    },
> > +    "report": {
> > +        "feed.code": {
> I think is not needed anymore
> 

Other CERTs use a Feed Code to anonymize the feed reference. I'd keep it. It is optional anyway. 
> —
> Reply to this email directly or view it on GitHub.
>